packages/openapi3/test/examples.test.ts

```diff
@@ -234,4 +234,385 @@ worksFor(["3.0.0", "3.1.0"], ({ openApiFor }) => {
       age: 2,
     });
   });
+
+  describe("parameters", () => {
+    it.each([
```

Comment:
can each of the it.each be grouped under a describe with the current test used in the it message and replace the message of the it just as the `desc` defined